app/pages/project/classify.cjsx

```diff
@@ -120,7 +121,7 @@ module.exports = React.createClass
         @setState {projectIsComplete}
         randomIndex = Math.floor Math.random() * workflows.length
         # console.log 'Chose random workflow', workflows[randomIndex].id
-        workflows[randomIndex]
+        @getWorkflow project, workflows[randomIndex].id
```

Comment:
Ah I see the addition of `fields` in the query above, so you'd have to fetch the one you want to get back the whole resource. Ignore my comment.